.circleci/test.sh

```diff
@@ -4,12 +4,16 @@ set -ex
 
 source /tmp/venv/bin/activate
 
+# install new libturbojpeg for pytorch/vision
+wget http://ftp.br.debian.org/debian/pool/main/libj/libjpeg-turbo/libturbojpeg0_2.0.5-1_amd64.deb
+sudo apt install ./libturbojpeg0_2.0.5-1_amd64.deb -y
+
 # install torchvision from master
 # the one on pypi requires cuda
 # Currently the master branch of pytorch/vision is not stable
```

Comment:
do comments on line 13 and 14 make sense anymore?

Reply:
Removed. Thank you for the reminder.